code/code/misc/equip.cc

```diff
@@ -119,6 +119,12 @@ bool TBeing::isWieldingWeapon() {
 
 bool TBeing::canUseEquipment(const TObj* o, silentTypeT silent,
   wearKeyT key) const {
+  if (!o) {
```

Comment:
Makes sense to me

Reply:
Technically this function should be receiving a reference to a TObj instead of a pointer, because the function won't work if `o` is null and references can't be null. But fixing that would be a bigger project and out of scope of this PR.